Src/cmor_fortran_interface.f90

```diff
@@ -806,8 +806,8 @@ end function cmor_close_cff
   integer, parameter:: CMOR_EXIT_ON_MAJOR = 0
   integer, parameter:: CMOR_EXIT_ON_WARNING = 2
 
-  real,    parameter:: CMOR_VERSION = 2.7
-  real,    parameter:: CMOR_CF_VERSION = 1.4
+  real,    parameter:: CMOR_VERSION = 3.7
+  real,    parameter:: CMOR_CF_VERSION = 1.6
```

Comment:
@mauzey1 the latest CF conventions are up to 1.9 - see http://cfconventions.org/Data/cf-conventions/cf-conventions-1.9/cf-conventions.html.

@taylor13 is there a reason that we roll back a few versions to 1.6 (or 1.4) here?

Reply:
@durack1 I wasn't sure about the CF version so I just used the one that was in cmor.h.

Reply:
For CMIP6 we have CF-1.7 in the "Conventions" attribute in the MIP table "Header"s, and only the information in the MIP tables propagates to the output as far as I can see.  I haven't kept track of CF standards changes and while I'd be surprised to see anything I'd be tempted to stick with the value used for CMIP6 here.

Reply:
@durack1 @taylor13 Should we update the CF version in CMOR to that of the CMIP6 tables?

Reply:
@mauzey1 yes please, this will be a good way to sync the CMIP6 tables and CMOR info, thanks